src/Dhadgar.ShoppingCart/src/components/auth/OAuthButtonGroup.tsx

```
@@ -26,59 +113,60 @@ export default function OAuthButtonGroup({
+              'relative flex items-center justify-center px-3 py-3',
+              'rounded-lg border border-glow-line/50 bg-panel-dark',
+              'transition-all duration-200',
+              'hover:border-text-secondary hover:bg-glow-line/20',
+              'focus:outline-none focus:ring-2 focus:ring-cyber-cyan/50',
+              'disabled:opacity-50 disabled:cursor-not-allowed',
+              loadingProvider === provider.id && 'animate-pulse'
+            )}
+            title={provider.name}
+          >
+            {loadingProvider === provider.id ? (
+              <svg className="animate-spin h-5 w-5" fill="none" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)